frontend/src/app/highlight/highlight.component.ts

```diff
@@ -9,13 +11,9 @@ import {AlertService} from '../alert.service';
   styleUrls: ['./highlight.component.css']
 })
 export class HighlightComponent implements OnInit {
-  alert: Alert;
+  alert$: Observable<Alert> = this.alertService.getMessage().pipe(shareReplay());
 
   constructor(private alertService: AlertService) {}
 
-  ngOnInit() {
-    this.alertService.getMessage().subscribe(alert => {
-      this.alert = alert;
-    });
-  }
+  ngOnInit() {}
```

Comment:
Sounds like we can get rid of this definition altogether, is that true?

Reply:
sure, and the import to